locales/pt-BR/translation.json

```
@@ -91,47 +91,47 @@
         "yes-please": "Sim, por favor"
     },
     "$Calendar": {
-        "leave-by": "Based on the time you arrived today, you should leave by",
-        "all-done": "All done for today. Balance of the day:",
-        "last-day-balance": "Last day used for balance",
+        "leave-by": "Com base na hora em que você chegou hoje, você deve sair às",
+        "all-done": "Tudo certo por hoje. Balanço do dia:",
+        "last-day-balance": "Último dia usado para balanço",
         "on": "On",
-        "working-days": "Working days",
-        "working-days-title": "How many working days there's in the month",
-        "month-balance": "Month Balance",
-        "month-balance-title": "Balance up until today for this month. A positive balance means extra hours you don't need to work today (or the rest of the month).",
-        "overall-balance": "Overall Balance",
-        "overall-balance-title": "Overall balance until end of the month or current day",
-        "switch-view": "Switch View",
-        "switch-view-title": "Switch View",
-        "current-month": "Current Month",
-        "current-month-title": "Go to Current Month",
-        "previous-month": "Previous Month",
-        "next-month": "Next Month",
+        "working-days": "Dias úteis",
+        "working-days-title": "Quantos dias úteis existem no mês",
+        "month-balance": "Balanço do Mês",
+        "month-balance-title": "Atual balanço para este mês. Um saldo positivo significa horas extras em que você não precisa trabalhar hoje (ou no restante do mês).",
+        "overall-balance": "Balanço Geral",
+        "overall-balance-title": "Balanço geral até o fim do mês ou dia atual",
+        "switch-view": "Mudar Visualização",
+        "switch-view-title": "Mudar Visualização",
```

To match the translation for the `FlexibleDayCalendar`, I think it would be better:

```suggestion
        "switch-view": "Trocar Visualização",
        "switch-view-title": "Trocar Visualização",
```